README.md

```diff
@@ -72,18 +72,17 @@ you declared in the igress.yaml in your `notes-app` repository, e.g. `http://<gi
 ## Step 8: Monitoring
 Monitoring of important metrics and logs is essential to successfully implement DevOps. Developers and other stakeholders must be able to observe the metrics in specialized dashboards, and receive notifications when certain metrics exceeds a defined threshold or if a failure switch has been triggered. Good
 dashboards can provide developers with important insight into parameters such as performance, usage patterns, resource capacity, stability during peak hours
-etc. There are many tools that one can use to set up monitoring with, and we will cover the basics of one such toolset, namely Grafana and Prometheus. 
+etc. There are many tools that one can use to set up monitoring with, and we will cover the basics of one such toolset, namely Grafana and Prometheus.
 Grafana and Prometheus are very popular, and the go to solution for many Kubernetes DevOps-driven environments. We will not cover monitoring of logs in this
 workshop.
 
 We have already installed Prometheus and Grafana in the workshop Kubernetes cluster at [grafana.devops.rosbach.no](grafana.devops.rosbach.no). The workshop
-instructors should already have provided you with the necessary credentials. The `notes-api` service exposes metrics to Prometheus using 
-[micrometer](micrometer.io), and the deployment manifest includes annotations that tell Prometheus where to scrape those metrics from. 
+instructors should already have provided you with the necessary credentials. The `notes-api` service exposes metrics to Prometheus using
+[micrometer](micrometer.io), and the deployment manifest includes annotations that tell Prometheus where to scrape those metrics from.
 
 ## Create a Dashboard
 It's time to create your very own insightful dashboard. We want our dashboard to show us the average number of requests per second in 5 minute windows.
-To create the dashboard navigate to [grafana.devops.rosbach.no](grafana.devops.rosbach.no), click on the `+` button on the left-hand side, and then click 
-on `Create`.
+To create the dashboard navigate to [grafana.devops.rosbach.no](http://grafana.devops.rosbach.no/), log in with the username "admin" (ask one of the workshop hosts for the super-secret password), click on the `+` button on the left-hand side, and then click on `Create`.
```

Comment:
Without "http://", GitHub would treat this url as relative to the github repo url.